src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574661,18 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040030
+name: GBA1-related Parkinson disease, susceptibility
+def: "Increased susceptibility for developing Parkinson disease due to the presence of a monoallelic pathogenic variant, likely pathogenic variant, or risk allele in the GBA1 gene." [https://orcid.org/0000-0003-3254-7564, https://search.clinicalgenome.org/CCID\:004931, PMID:19433656, PMID:19846850, PMID:19858467]
+is_a: MONDO:0020573 {source="https://orcid.org/0000-0002-0587-4693"} ! inherited disease susceptibility
+intersection_of: MONDO:0020573 {source="https://orcid.org/0000-0002-0587-4693"} ! inherited disease susceptibility
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4177 {source="https://orcid.org/0000-0002-0587-4693"} ! GBA1
+intersection_of: predisposes_towards MONDO:0005180 {source="https://orcid.org/0000-0002-0587-4693"} ! Parkinson disease
+relationship: has_characteristic MONDO:0021152 {source="https://orcid.org/0000-0002-0587-4693"} ! inherited
+property_value: http://purl.org/dc/elements/1.1/date "2024-07-02T11:34:09Z" xsd:dateTime
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+
```

Comment:
please add the term tracker item to https://github.com/monarch-initiative/mondo/issues/7859

see: https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#term-tracker-item

Reply:
Comment addressed!